portfolio/src/main/webapp/index.html

```diff
@@ -49,53 +50,148 @@ <h4>Software Engineering Intern</h4>
     </div>
 
     <!-- Main Content Section -->
+
     <div id="home" class="menu">
-      <section>
-          <div class="info">
-            <div class="title"><h2>About Me</h2></div>
-              <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
-              </p>
+      <section id="home" class="landing">
+        <div class="content"> 
+          <h2>Hi there.<br> I'm <span> Kayla Leung </span></h2>
+          <button id="testbutton" class="button" onclick="document.location = '#aboutme'">About Me</button>
+        </div>
+      </section>
+    </div>
+
+    <div id="aboutme" class="menu">
+      <section class="banner">
+        <div class="info">
+          <div class="title">
+            <h1>About Me</h1>
+          </div>
+            <p>
```

Comment:
Your actual paragraph doesn't start until the text, so that's where this should go. In fact, in the browser I see Firefox has closed the p tag straight away and opened a new one later in an incorrect attempt to fix this HTML.